src/OptinForms/AbstractOptinTheme.php

```diff
@@ -1121,6 +1121,44 @@ public function shortcode_optin_form_custom_fields($atts)
                             }
                             $html .= "</select>" . $atts['tag_end'];
                             break;
+                        case 'country':
+                            $display_type = moVar($field, 'list_country_options_field_options', 'alpha_two', true);
+
+                            $alignment_style = sprintf("text-align: %s;", moVar($field, 'list_contries_alignment', 'left', true));
+                            $style           .= $alignment_style;
+
+                            $html .= $atts['tag_start'];
+
+                            switch($display_type) {
+                                case 'alpha_two':
+                                    $countries_lists =  \MailOptin\Core\countries_array();
+                                    if(is_array($countries_lists) && !empty($countries_lists)) {
+                                        $placeholder = ! empty($placeholder) ? $placeholder : esc_html__('Select...', 'mailoptin');
+                                        $html        .= "<select name=\"$field_id\" $data_attr class=\"$class\" id=\"$id\" style=\"$style\">";
+                                        $html        .= "<option value='' selected='selected'>$placeholder</option>";
+
+                                        foreach ($countries_lists as $index => $countries_list) {
+                                            $html .= sprintf('<option value="%s (%s)">%s</option>', $countries_list, $index, $index);
+                                        }
+                                        $html .= '</select>';
+                                    }
+                                break;
+                                case 'alpha_three':
+                                    $countries_lists =  \MailOptin\Core\countries_array('alpha_three');
+                                    if(is_array($countries_lists) && !empty($countries_lists)) {
+                                        $placeholder = ! empty($placeholder) ? $placeholder : esc_html__('Select...', 'mailoptin');
+                                        $html        .= "<select name=\"$name_attribute\" $data_attr class=\"$class\" id=\"$id\" style=\"$style\">";
+                                        $html        .= "<option value='' selected='selected'>$placeholder</option>";
+
+                                        foreach ($countries_lists as $index => $countries_list) {
+                                            $html .= sprintf('<option value="%s (%s)">%s</option>', $countries_list, $index, $index);
```

Comment:
And I think you got the sprinf parameters wrong. Here is the output

![image](https://user-images.githubusercontent.com/4140659/89864527-fc768780-dba3-11ea-978f-982dcf91a89d.png)

Reply:
What should be the output? I was thinking of putting the full country as value and then the selected option inside a bracket. What do you think?

Reply:
The alpha codes should be the country value. while the country name and alpha code in bracket the label
The inverse is currently the case.